admin/data/sql/migrations/20200210153747.undo.update-sp-create-checks-pt-2.sql

```diff
@@ -0,0 +1,2 @@
+EXEC sp_rename 'mtc_admin.checkPin.pinExpiresAtUtc', pinExpiresAt, 'COLUMN';
+ALTER TABLE mtc_admin.checkPin DROP COLUMN pinValidFromUtc;
```

Comment:
The undo here does not re-create the spCreateChecks proc which it must.

What it ought to do here is:

1) Drop and recreate the TVP
2) Drop and recreate the SP spCreateChecks

These ought to be separate migrations.